docs/jwst/introduction.rst

```diff
@@ -387,6 +392,22 @@ Alternatively you can specify the ``skip`` argument on the command line:
 Logging Configuration
 ---------------------
 
+The name of a file in which to save log information, as well as the desired
+level of logging messages, can be specified in an optional configuration file
+"stpipe-log.cfg". This file must be in the same directory in which you run the
+pipeline in order for it to be used. If this file does not exist, the default
+logging mechanism is STDOUT, with a level of INFO. An example of the contents
+of the stpipe-log.cfg file is:
+::
+
+    [*]
+    handler = file:pipeline.log
+    level = INFO
+
+which specifies that all log messages will be directed to a file called
+"pipeline.log" and messages at a severity level of INFO and above will be
+recorded.
+
 If there's no ``stpipe-log.cfg`` file in the working directory, which specifies
```

Comment:
The first sentence is now redundant.

It would be useful to point out that there is a very big advantage to using a log configuration file with a non-default file name.  If you use the default name and then a week later forget that stpipe-log.cfg is in the directory, your log output will disappear "into the void" (I'm writing from experience).  If you use a different name, it's safe to forget about it because it won't be used by default.